src/applications/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -25,6 +25,22 @@ class FacilityDetail extends Component {
     focusElement('.va-nav-breadcrumbs');
   }
 
+  componentDidUpdate(prevProps) {
+    const justLoaded =
+      prevProps.currentQuery.inProgress && !this.props.currentQuery.inProgress;
+
+    if (justLoaded) {
+      this.__previousDocTitle = document.title;
+      document.title = `${
+        this.props.facility.attributes.name
+      } | Veterans Affairs`;
+    }
+  }
+
+  componentWillUnmount() {
+    document.title = this.__previousDocTitle;
+  }
```

Comment:
I don't really like restoring the original doc title like this, but it seems like the simplest way. @U-DON what do you think?

Reply:
I'm fine with the simple approach. This still works well since you can't go to any other routes from this facility view, so the title change seems to be self contained.